core/include/detray/propagator/actors/aborters.hpp

```diff
@@ -58,7 +58,8 @@ struct pathlimit_aborter : actor {
         // Check the path limit
         if (step_limit <= 0.f) {
             // Stop navigation
-            prop_state._heartbeat &= nav_state.abort();
+            prop_state._heartbeat &=
+                nav_state.abort("Aborter: Maximal path length reached");
```

Comment:
Is there a reason these are strings and not enums? Especially if this is all defined in detray?

Reply:
Isn't that just a more complicated way of achieving the same thing, given that atm I just need to know which string to print (and there is only one choice here)? 
Not all aborters will be defined in detray (e.g. ckf and Kalman step aborters live in traccc), so I cannot add their error status codes to the navigator up front unfortunately